### PR TITLE
Code / Project cleanup to reduce number of warnings

### PR DESCRIPTION
**Pull request by daalta**
### Purpose
* #2526 requests a reduction in the number of editor warnings.
* This PR reduces the number of warnings generated by starting the lobby scene to 0.
* Compiling still causes over 400 warnings. But some have been removed.
* There still **much** left to be done before #2526 is complete.

Also...

* Removed bedsheet placed at the captain's spawn (https://github.com/unitystation/unitystation/pull/2707 )

### Notes
* Removed some unused imports
* Deleted empty prefabs
* Removed components with missing script references
* Unreachable code was...
  * Commented out if it will be needed in the future
  * Removed if it was not needed
  * Left as it is with a TODO if unsure.
* Automformatted some files
### Checks

- [x] The PR does not include any unnecessary .meta, .prefab or <b>.unity (scene) changes</b>
- [x] The PR does not bring up any new compile errors
- [x] The PR has been tested in editor

Tests were not extensive as behaviour should be unchanged.

### CODE REVIEW

File: UnityProject/Assets/Scripts/FloorDecals/FloorDecal.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public override void OnStartClient()
 
 	private void SyncChosenSprite(int _chosenSprite)
 	{
-		chosenSprite = chosenSprite;
+		chosenSprite = chosenSprite; // TODO Fix this compiler warning or remove this line.
```
Comment (DooblyNoobly) [inline UnityProject/Assets/Scripts/FloorDecals/FloorDecal.cs]: what the! I think this was meant to be chosenSprite = _chosenSprite

Comment (daalta) [reply UnityProject/Assets/Scripts/FloorDecals/FloorDecal.cs]: Thought the same thing, but I'm not familiar with how floor decals are supposed to work.

Would you like me to change or remove the line?

**Comment by daalta** [comment]
(Oops, pushed that one to the wrong branch.)

File: UnityProject/Assets/Scripts/FloorDecals/FloorDecal.cs (hunk 1 above)
Comment (DooblyNoobly) [reply UnityProject/Assets/Scripts/FloorDecals/FloorDecal.cs]: yep if you can make the change I'll merge this

**Pull request merged**